### PR TITLE
IDE-4659 update modules ext files wizard test and sf

### PR DESCRIPTION
**Pull request by rui123wang**
IDE-4659 update modules ext files wizard test and sf

### CODE REVIEW

File: functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/action/WizardAction.java [hunk 1]
```diff
@@ -797,6 +801,12 @@ public Text origialModule() {
 			return _newModulesExtFilesWizard.getOrigialModule();
 		}
 
+		public void prepareModuleExtProjectName(String projectName) {
```
Comment (yanan) [inline functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/action/WizardAction.java]: 1. rename to selectModuleExtProjectName

File: functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/action/WizardAction.java [hunk 2]
```diff
@@ -797,6 +801,12 @@ public Text origialModule() {
...
+			moduleExtprojectName().setSelection(projectName);
+
+			ide.sleep();
```
Comment (yanan) [inline functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/action/WizardAction.java]: 2. can we not use sleep here?

File: functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/page/wizard/project/NewLiferayModulesExtWizard.java [hunk 3]
```diff
@@ -44,4 +49,8 @@ public Text getOrigialModuleVersion() {
 		return new Text(getShell().bot(), ORIGINAL_MODULE_VERSION);
 	}
 
+	public void selectLaunchModulesExtFiles() {
```
Comment (yanan) [inline functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/page/wizard/project/NewLiferayModulesExtWizard.java]: 3. remove this duplicate method

File: functional-tests/tools/modules.ext/modules.ext.files.wizard/modules.ext.files.wizard.create/src/com/liferay/ide/functional/modules/ext/files/wizard/tests/NewModulesExtFilesTests.java [hunk 4]
```diff
@@ -52,6 +50,8 @@ public void addModulesExtWithJava() {
 
 		wizardAction.newModulesExtFiles.openFileMenuModulesExtFilesWizard();
 
+		wizardAction.newModulesExtFiles.prepareModuleExtProjectName(project.getName());
```
Comment (yanan) [inline functional-tests/tools/modules.ext/modules.ext.files.wizard/modules.ext.files.wizard.create/src/com/liferay/ide/functional/modules/ext/files/wizard/tests/NewModulesExtFilesTests.java]: 4. change one case to use Launch New way

File: functional-tests/tools/modules.ext/modules.ext.files.wizard/modules.ext.files.wizard.create/src/com/liferay/ide/functional/modules/ext/files/wizard/tests/NewModulesExtFilesTests.java [hunk 5]
```diff
@@ -52,6 +50,8 @@ public void addModulesExtWithJava() {
...
+
```
Comment (yanan) [inline functional-tests/tools/modules.ext/modules.ext.files.wizard/modules.ext.files.wizard.create/src/com/liferay/ide/functional/modules/ext/files/wizard/tests/NewModulesExtFilesTests.java]: 5 add one test case to create empty ext files > finish

File: functional-tests/tools/modules.ext/modules.ext.files.wizard/modules.ext.files.wizard.validation/src/com/liferay/ide/functional/modules/ext/files/wizard/tests/ValidationModulesExtFilesTests.java [hunk 6]
```diff
@@ -95,7 +93,7 @@ public void checkExtFilesWithoutProjects() {
 
 		validationAction.assertEnabledTrue(wizardAction.getBackBtn());
 
-		validationAction.assertEnabledTrue(wizardAction.getFinishBtn());
+		validationAction.assertEnabledFalse(wizardAction.getFinishBtn());
```
Comment (yanan) [inline functional-tests/tools/modules.ext/modules.ext.files.wizard/modules.ext.files.wizard.validation/src/com/liferay/ide/functional/modules/ext/files/wizard/tests/ValidationModulesExtFilesTests.java]: 6. add one test case to test module ext project name
mainly on: when no ext project, check available project name
create two ext project, ext files wizard check available project names

**Comment by yanan** [comment]
Please see the comments to update and resend, thanks
